article/_posts/2022-04-14-MultiModalVersatileNetworks.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: review
+title: "Self-Supervised MultiModal Versatile Networks"
+tags: deep-learning representation-learning self-supervised multi-task
+author: "Nathan Painchaud"
+cite:
+    authors: "Jean-Baptiste Alayrac, Adria Recasens, Rosalia Schneider, Relja Arandjelović, Jason Ramapuram, Jeffrey De Fauw, Lucas Smaira, Sander Dieleman, Andrew Zisserman"
+    title:   "Self-Supervised MultiModal Versatile Networks"
+    venue:   "NeurIPS 2020"
+pdf: "https://proceedings.neurips.cc/paper/2020/file/0060ef47b12160b9198302ebdb144dcf-Paper.pdf"
+---
+
+
+# Highlights
+- Introduce the notion of a MultiModal Versatile (MMV) network, that can ingest multiple modalities and outputs
+representations useful for downstream tasks in multiple modalities;
+    - The papers especially studies how best to combine the modalities, so that representations respect four properties
+- Introduce the process of deflation, to efficiently apply networks on video data to static images.
+
+
+# Introduction
+The authors' motivation is to propose a network for joint video-audio-text representation that respects four properties:
+1. Able to take as input any of the three modalities;
+2. Respect the specificity, i.e. dimensionality, of the modalities: audio and video are much finer than text;
```

Comment:
Not sure that is meant by "finer"

Reply:
Basically, this is the term used by the author to mean that video/audio data is high-dimensional and contains a lot of details, by contrast to the limited, high-level information provided by text data. I included this explanation in the review itself so that it is clear for everyone.